site/payment-gateway/v2/pay.md

```diff
@@ -0,0 +1,102 @@
+---
+
+---
+
+# Payment Gateway: Pay
+
+[API Docs](/) > [Payment Gateway](/payment-gateway/) > [Pay](/payment-gateway/v2/pay)
+
+## Payment Request
+
+This endpoint will take payment using a token and required payment parameters.
+
+**Note**: Token payment is currently available for Level 5 Germany agents only.
+
+### Method
+
+POST
+
+### Endpoint
+
+To make a payment, the endpoint to use is:
+
+```
+https://payment-gateway.holidayextras.co.uk/pay
+```
+
+### Request Headers
+
+| Name  | Data Type | Format | Mandatory? | Additional Information |
+| ----  | --------- | ------ | ---------- | ---------------------- |
+| Content-Type | String | `application/json` | Y | This should always be `application/json` |
+| Authorization   | String | `Basic dXNlcm5hbWU6cGFzc3dvcmQ=` | Y | Base64 encoded string for the `username:password` credentials which will be created and provided to you. This header is required with every request.|
+| Accept | String | `application/vnd.holidayextras.v2+json` | N | To use a different version please provide a different value. |
+
+### Request Parameters
+
+NB: All parameter names are case sensitive.
+
+ | Name  | Data Type | Format | Mandatory? | Additional Information |
+ | ----  | --------- | ------ | ---------- | ---------------------- |
+ | channel | String | [A-Z] | Y | This will be supplied with your authentication credentials. |
+ | paymentId | String | [0-9A-Z]{16} | N | A unique ID to be used when making the payment. This cannot already exist otherwise payment will fail. If this is not supplied, then one will be generated for this payment and returned as part of the result. |
+ | token   | String | [0-9] | Y | A tokenised card to make the payment against. This needs to exist already, otherwise payment will fail. |
+ | paymentReference | String | [A-Z] | Y | A reference used to group multiple payments together, doesn't need to be unique. |
+ | amount | String | [0-9] | Y | The price (in pence) to be paid. |
```

Comment:
This works as either a string or a number, the refund endpoint also documents a string but *requires* a number - these should both be swapped for numbers I think